cmd/gitmono/main.go

```diff
@@ -58,26 +64,33 @@ func main() {
 
 	// inject options to global component
 	flagsParser.CommandHandler = func(command flags.Commander, args []string) error {
+		// options are ready to use
+		log.SetOutput(ioutil.Discard)
+		if opts.Verbose {
+			log.SetFlags(log.Flags() &^ (log.Ldate | log.Ltime))
```

Comment:
Removing datetime from logs to enable cmdtest for logs. Still considering the idea of logs testing :)